### PR TITLE
LPS-190283 EVP Automated Test | Master

### PR DESCRIPTION
**Pull request by joelrocha68**
[LPS-190284](https://liferay.atlassian.net/browse/LPS-190284)
[LPS-190285](https://liferay.atlassian.net/browse/LPS-190285)
[LPS-190286](https://liferay.atlassian.net/browse/LPS-190286)

### CODE REVIEW

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:osb-site-initializer-evp

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequestForm.testcase [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	}
+
+	@description = "LPS-162902 - Verify that can cancel the request creation"
+	@priority = 3
+	test CanCancelRequestCreation {
+		task ("When the user cancel a request form") {
+			Button.clickCancel();
+		}
+
+		task ("The user is then redirected to the request listing page") {
```
Comment (MGabriellaS) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequestForm.testcase:57]: I think you  can change this task  for: `"Then the user can see that there were no requests being showed"`

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by MGabriellaS** [comment]
The code looks good to me!

**Comment by MGabriellaS** [comment]
ci:forward

**Pull request closed without merging**